File: SofiForce.Models/Models/EntityModels/PromotionGroupModel.cs
using System;

namespace Models
{
    public class PromotionGroupModel
    {
        public int PromotionGroupId { get; set; }
        public string PromotionGroupCode { get; set; }
        public string PromotionGroupNameEn { get; set; }
        public string PromotionGroupNameAr { get; set; }
        public bool? IsActive { get; set; }
        public bool? CanEdit { get; set; }
        public bool? CanDelete { get; set; }
        public string Color { get; set; }
        public string Icon { get; set; }
        public int? DisplayOrder { get; set; }
        public int? CBy { get; set; }
        public DateTime? CDate { get; set; }
        public int? EBy { get; set; }
        public DateTime? EDate { get; set; }
    }
}

File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/PromotionGroupModel.ts
export interface PromotionGroupModel {
    promotionGroupId: number;
    promotionGroupCode: string;
    promotionGroupNameEn: string;
    promotionGroupNameAr: string;
    isActive: boolean | null;
    canEdit: boolean | null;
    canDelete: boolean | null;
    color: string;
    icon: string;
    displayOrder: number | null;
    cBy: number | null;
    cDate: Date | null;
    eBy: number | null;
    eDate: Date | null;
}